### PR TITLE
Pass region and AWS credentials with AWS API requests

### PR DESCRIPTION
**Pull request by danotorrey**
The `AWSResource.getLogGroupNames` and `AWSResource.getKinesisStreams` API resource methods now require a POST body with region and AWS credentials. Before, these methods were using implicit environment variable credentials.

### CODE REVIEW

**Comment by danotorrey** [comment]
Resolves #87

File: src/main/java/org/graylog/integrations/aws/resources/AWSResource.java [hunk 1]
```diff
@@ -55,36 +57,62 @@ public AWSResource(AWSService awsService, KinesisService kinesisService, CloudWa
...
+     * --data-binary '{
+     * "region": "us-east-1",
+     * "aws_access_key_id": "some-key",
+     * "aws_secret_access_key": "some-secret"
+     * }'
+     */
+    @POST
     @Timed
-    @Path("/cloudWatch/logGroups/{regionName}")
+    @Path("/cloudWatch/logGroups")
```
Comment (bernd) [inline src/main/java/org/graylog/integrations/aws/resources/AWSResource.java]: We usually use `snake_case` for API paths.

File: src/main/java/org/graylog/integrations/aws/resources/AWSResource.java [hunk 2]
```diff
@@ -55,36 +57,62 @@ public AWSResource(AWSService awsService, KinesisService kinesisService, CloudWa
...
     @Path("/regions")
     @ApiOperation(value = "Get all available AWS regions")
-    public List<RegionResponse> getAwsRegions() {
+    public List<RegionsResponse> getAwsRegions() {
         return awsService.getAvailableRegions();
     }
 
     @GET
     @Timed
     @Path("/availableServices")
```
Comment (bernd) [inline src/main/java/org/graylog/integrations/aws/resources/AWSResource.java]: We usually use `snake_case` for API paths.

**Comment by danotorrey** [comment]
Resolves #105

**Pull request merged**